### DISCUSSION TITLE
Dependabot alert on a non-existing path / file

### DISCUSSION THREAD

**Discussion by assayire** [community/community#22521] [Other Feature Feedback, Questions, & Ideas]
<p>I am getting the following dependabot <a href="https://github.com/vivekragunathan/vivekragunathan.github.io/security/dependabot" rel="noopener nofollow ugc"><strong>alert</strong></a> in my github pages repo.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/1/b14b310d379aad136c573c2c12a5af1ad6c30533.png" data-download-href="/uploads/short-url/pipBN5ZQ7NCuZ49KhnP4x2vz9lx.png?dl=1" title="CleanShot 2022-02-07 at 16.57.11@2x" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/66568479/181071587-b20577c6-cb8b-4756-a169-fad52e492f18.png" alt="CleanShot 2022-02-07 at 16.57.11@2x" data-base62-sha1="pipBN5ZQ7NCuZ49KhnP4x2vz9lx" width="690" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b14b310d379aad136c573c2c12a5af1ad6c30533_2_690x499.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b14b310d379aad136c573c2c12a5af1ad6c30533_2_1035x748.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b14b310d379aad136c573c2c12a5af1ad6c30533_2_1380x998.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b14b310d379aad136c573c2c12a5af1ad6c30533_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">CleanShot 2022-02-07 at 16.57.11@2x</span><span class="informations">1896×1372 244 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<ol>
<li>I don’t have that path - <code>/docs/blog/themes/tranquil-peak/..</code> referred by depenadabot anywhere in my repo / branches. It was a theme I was trying a long time back.</li>
<li>There is an option to create a dependabot update that should fix this alert automatically. But clicking the button just sits there, nothing happens.</li>
</ol>
<p>How do I fix or get rid of the alert? Perhaps by removing some or all of the previous deployments? If so, is there a way to remove deployments?</p>
<p>Help much appreciated.</p>

**Comment by yoannchaudet** [comment]
<p>Hello!</p>
<p>It looks like it was stale data. I got someone on the dependabot team to clear that for you.</p>
<p>You should still see the vulnerability but in closed state not opened like it was before.</p>
<p>Let me know if that helps</p>

**Comment by assayire** [comment]
<p>Oh wow thanks a lot man. Yes i see it in a closed state.</p>

**Comment by RobThree** [comment]
I just got a notification of several dependabot alerts about files that don't exist...

Repo | Message | File
-- | -- | --
https://github.com/RobThree/ThankYouVeryMuch | Cross-Site Scripting (XSS) in jquery | Detected in jquery (npm) • js/jquery-1.10.2.min.js
https://github.com/RobThree/RobIII | Cross-Site Scripting (XSS) in jquery | Detected in jquery (npm) • RobIII/js/vendor/jquery-2.2.2.min.js
https://github.com/RobThree/Luca | Cross-Site Scripting (XSS) in jquery | Detected in jquery (npm) • js/vendor/jquery-1.11.3.min.js
https://github.com/RobThree/Luca | Cross-Site Scripting (XSS) in jquery | Detected in jquery (npm) • js/vendor/jquery-2.1.4.min.js
https://github.com/RobThree/DeKnusseKnipperette | Cross-Site Scripting (XSS) in jquery | Detected in jquery (npm) • js/vendor/jquery-1.11.3.min.js
https://github.com/RobThree/DeKnusseKnipperette | Cross-Site Scripting (XSS) in jquery | Detected in jquery (npm) • js/vendor/jquery-2.1.4.min.js

None of these files exist. Any chance you can have someone look into this too @yoannchaudet ?

**Reply by yoannchaudet** [reply]
I am told support has a process to re-sync something in your repos. Drop them a [message](https://support.github.com/contact)!
